apps/pdc-dashboard/config/sync/admin-role.pdc-lq539osz.json

```diff
@@ -344,9 +344,7 @@
           "metadata.attachementAdded",
           "metadata.contact.name",
           "metadata.contact.email",
-          "metadata.identificationMethod.method",
           "metadata.eForm.linkedEForm",
```

Comment:
Waarvoor is de linkedEForm property nog wel nodig?

Reply:
Deze aanvraag kwam van het Data Analist team om het volgen van producten en bijbehorende formulieren mogelijk te maken, evenals de gebruikte logging methode. Daarnaast, na meerdere vergaderingen met Lauren van het Data Analist team, informeerde ze me dat dit veld overbodig is. Ze kunnen de benodigde informatie ophalen vanuit de CTA-knop. In de toekomst zal er een functie zijn waarbij een knop alle vereiste informatie zal ophalen en downloaden als een CSV-bestand.

Reply:
Nee, nog niet gevraagd om te verwijderen

Reply:
Ik ga ervan uit dat deze data nog steeds dubbel is. Het wel of niet bestaan van een eform is dan ook te herleiden uit de CTA knop, maar dat horen we dan nog wel.

Reply:
Sorry, mijn eerste opmerking verwijst naar `"metadata.identificationMethod.method"` en niet naar je vraag. Ik denk dat het `linkedEForm` veld bedoeld is voor het data-analistenteam om rapporten te kunnen maken. Misschien wordt dit veld later verwijderd, denk ik.